src/api/models.py

```diff
@@ -22,81 +22,117 @@
 from sqlmodel import Field, SQLModel, Relationship
 
 
+
 class SignalModel(SQLModel, table=True):
     __tablename__ = "signals"
 
+    context: Dict = Field(
+        sa_column=Column(JSONB),
+        default={},
+        description="Context mapping vocabulary to IRIs",
+        alias= "context_")
+    
+    type: str = Field(sa_column_kwargs = {"server_default": "dcd:Dataset"},
```

Comment:
Typo here? "dcd" -> "dct"

Reply:
This is not a typo. 
I used dcd as the acronym for path http://purl.org/dc/dcmitype, which is where Dataset exists "http://purl.org/dc/dcmitype/Dataset". 
while I used dct as acronym for  http://purl.org/dc/terms, which is where the terms exist. 
I also just found out that Dataset is in that path

Reply:
Would it be better to use the definition for Dataset from dcat? Ultimately, we should have Alejandra look at these, but that might not happen that soon.

Reply:
Absolutely. I have just changed it to dcat.